alphastats/gui/pages/02_Import Data.py

```diff
@@ -1,14 +1,21 @@
 import streamlit as st
```

Comment:
probably best to not look at the diff for this page, but just at the new version